pubtools/_pulp/tasks/push/command.py

```diff
@@ -174,14 +186,24 @@ def uploaded_items(self, items):
 
                 needs_upload.append(item.ensure_uploaded(ctx))
 
-        LOG.info(
-            "Upload: %s item(s) already present, %s uploading",
-            len(uploaded),
-            len(needs_upload),
-        )
+        event = {
+            "type": "uploading-pulp",
+            "items-present": len(uploaded),
+            "items-uploading": len(needs_upload),
+        }
+        messages = [
+            "%s already present" % len(uploaded),
+            "%s uploading" % len(needs_upload),
+        ]
 
-        # Anything already in the system can be immediately yielded.
-        for item in uploaded:
+        if self.args.pre_push:
+            messages.append("%s skipped during pre-push" % len(prepush_skipped))
+            event["items-prepush-skipped"] = len(prepush_skipped)
+
+        LOG.info("Upload items: %s", ", ".join(messages), extra={"event": event})
```

Comment:
This introduces some inconsistency in log message format. Are we to assume Update will get the same treatment?

Reply:
Yes, it'd be a good idea to make them consistent in a later patch.